entity-framework/core/modeling/dynamic-model.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Dynamic Models - EF Core
```

Comment:
@AndriySvyryd @ajcvickers I am a bit concerned that the term "Dynamic" may convey magic capabilities that we don't have, but I am struggling to come up with something better for the title, e.g. "Changing the model at runtime" isn't really more constrained, or for the class name, e.g. `ParameterizedDbContext` isn't great. Any better ideas? If it is too hard I can settle for "dynamic".

Reply:
Maybe something along the lines of, "Using multiple models with the same context type"? Or "One DbContext with many models"?

Reply:
I like that. It can be read as using all the models at the same time, but not sure it is a major issue... Anyway, what about "Alternating between multiple models with the same DbContext type"?

Reply:
Sounds good.